apps/web-giddh/src/app/vouchers/template/template-edit-filter/template-edit-filter.component.html

```
@@ -1,3 +1,10 @@
+@let isTemplateA = customTemplate?.templateType === templateTypeEnum.GstTemplateA;
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)